tests/wpunit/Admin/Fields/FieldTest.php

```diff
@@ -266,7 +266,7 @@ public function it_should_render_correct_html_for_oauth_resource_with_license_ke
 		update_option( 'test_storage', [
 			'stellarwp_auth_url_service_oauth_with_license_key_field_1' => [
 				'expiration' => 0,
-				'value'      => 'https://licensing.stellarwp.com/account-auth?uplink_domain=&uplink_slug=service-oauth-with-license-key-field-1&_uplink_nonce=535281edcd',
```

Comment:
I'm curious why this is being removed? Was the test passing before? Did something change to actually not require those query args? Were they useless before and this is cleanup?

Reply:
Yep, they were unnecessary when the test was first added. Those values are combined and encoded for the `uplink_callback` parameter here in `Auth_Url_Builder::build()`: https://github.com/stellarwp/uplink/blob/11b5e6d5d7db323f2026dcf20a397b574a8b8e3a/src/Uplink/Auth/Auth_Url_Builder.php#L59-L80

Reply:
I had to update this because of the new data attribute, so I made some changes there too.